Enhancement/scan input autofocus

ikusteu opened the pull request:
Fixes #318

silviot commented on pkg/ui/src/Forms/ScanInput/action.ts:
```diff
@@ -0,0 +1,95 @@
+			}
+
+			const buffer = state.buffer + e.key;
+			const timeout = setTimeout(store.cancel, 50);
```
@ikusteu To clarify, I don't mean it needs to be configurable via an environment variable. My point is just that it's an "important" value that might need some tweaking before finding the Goldilocks "Just right" value, and thus has dignity to have a name.

ikusteu replied:
Yeah, I got it. Makes sense...

ikusteu replied:
@silviot do you think I should put it next to the action (in the same folder as `constants.ts`) or should I have the action accept it as a parameter, and specify the constant, when consuming it - in the app?

silviot replied:
I think just having it defined in the same file, at the top of the file, makes it easier to find when needed.

ikusteu commented:
@silviot can I get a final amen on the PR, I've extracted the timeout into a variable.

_(I've rebased against `main`, including some flakiness fixes and other PRs merged in the past couple of hours, so the original comment is not referencing the current piece of code)_
